Buildings/Experimental/DHC/EnergyTransferStations/Cooling/Indirect.mo

```diff
@@ -0,0 +1,301 @@
+    final m_flow_nominal=mDis_flow_nominal)
+    "District supply temperature sensor"
+    annotation (Placement(transformation(extent={{-220,-290},{-200,-270}})));
+  Buildings.Fluid.Sensors.MassFlowRate senMasFlo(
+    redeclare final package Medium=MediumSer)
+    "District supply mass flow rate sensor"
+    annotation (Placement(transformation(extent={{-160,-290},{-140,-270}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage conVal(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=mDis_flow_nominal,
+    final dpValve_nominal=dpConVal_nominal,
+    riseTime(displayUnit="s") = 10,
+    y_start=0)
+    "Control valve"
+    annotation (Placement(transformation(extent={{-100,-290},{-80,-270}})));
+  Buildings.Fluid.HeatExchangers.PlateHeatExchangerEffectivenessNTU hex(
+    redeclare package Medium1 = MediumSer,
+    redeclare package Medium2 = MediumBui,
+    final m1_flow_nominal=mDis_flow_nominal,
+    final m2_flow_nominal=mBui_flow_nominal,
+    final dp1_nominal=dp1_nominal,
+    final dp2_nominal=dp2_nominal,
+    final configuration=Buildings.Fluid.Types.HeatExchangerConfiguration.CounterFlow,
+    final Q_flow_nominal=Q_flow_nominal,
+    final T_a1_nominal=T_a1_nominal,
+    final T_a2_nominal=T_a2_nominal)
+    annotation (Placement(transformation(extent={{0,-200},{20,-220}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTDisRet(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=mDis_flow_nominal)
+    "District return temperature sensor"
+    annotation (Placement(transformation(extent={{180,-270},{200,-290}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiRet(
+    redeclare final package Medium=MediumBui,
+    final m_flow_nominal=mBui_flow_nominal)
+    "Building return temperature sensor"
+    annotation (Placement(transformation(extent={{-218,210},{-198,190}})));
+  Modelica.Blocks.Interfaces.RealInput TSetBuiSup
+    "Setpoint temperature for building supply"
+    annotation (Placement(transformation(extent={{-340,-20},{-300,20}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiSup(
+    redeclare final package Medium=MediumBui,
+    final m_flow_nominal=mBui_flow_nominal)
+    "Building supply temperature sensor"
+    annotation (Placement(transformation(extent={{200,210},{220,190}})));
+  Buildings.Controls.Continuous.LimPID con(
+    final controllerType=Modelica.Blocks.Types.SimpleController.PID,
+    final k=k,
+    final Ti=Ti,
+    final Td=Td,
+    final yMax=yMax,
+    final yMin=yMin,
+    final wp=wp,
+    final wd=wd,
+    final Ni=Ni,
+    final Nd=Nd,
+    final initType=Modelica.Blocks.Types.Init.InitialOutput,
+    final xi_start=xi_start,
+    final xd_start=xd_start,
+    final y_start=yCon_start,
+    final reverseActing=reverseActing)
+    "Building supply temperature controller"
+    annotation (Placement(transformation(extent={{-180,10},{-160,-10}})));
+  Modelica.Blocks.Math.Add dTdis(
+    final k1=-1,
+    final k2=+1)
```

Comment:
no need to specify `k2`.

Reply:
Putting a "final" prefix for k2=+1 to keep k2 unchanged?